.github/CODEOWNERS

```diff
@@ -740,6 +728,18 @@
 /core/templates/services/auth-backend-api.service*.ts @oppia/lace-frontend-reviewers
 
 
+# Dynamic Feature Gating project
+/core/controllers/feature_flag*.py @oppia/core-reviewers
```

Comment:
Isn't there anything at all that needs to be assigned to the compatibility reviewers, like the configurable definitions of feature flags, platform params, etc.? (Android will add their own ones.) See the longer description in my earlier comment: https://github.com/oppia/oppia/pull/19632#discussion_r1479550450

Reply:
Ah! right, I have included the files for the web-android compatible reviewers. The rest of the files include the logics for platform parameters and feature flags, that I have assigned to core-reviewers.
Thanks